src/java.base/share/classes/java/io/SimpleIO.java

```diff
@@ -0,0 +1,1037 @@
+     * That perches in the soul
+     * And sings the tune without the words
+     * And never stops - at all
+     *
+     * }
+     *
+     * @implSpec Line terminators are normalized the platform line separator.
+     *
+     * @param path  path of file to be written
+     * @param content  string content of the file
+     * @throws UncheckedIOException wrapping an IOException if an io error occurs.
+     */
+    public static void write(Path path, String content) {
+        Objects.requireNonNull(path, "path must not be null");
+        Objects.requireNonNull(content, "content must not be null");
+        try (PrintWriter out = new PrintWriter(path.toFile())) {
+            content.lines().forEach(out::println);
+        } catch (IOException ex) {
+            throw new UncheckedIOException(ex);
+        }
+    }
+
+    /**
+     * Write a list of strings as the content of the file {@code filename}.
+     * For example:
+     * {@snippet lang="java":
+     * var supply = List.of("apple", "pear", "orange");
+     * writeLines("data.txt", supply); // @highlight substring="writeLines"
+     * var lines = readLines("data.txt");
+     * printlines(lines);
+     * }
+     * will print on the output stream as:
+     * {@snippet lang="text":
+     * apple
+     * pear
+     * orange
+     *
+     * }
+     *
+     * @implSpec Each line will be followed by the platform line separator.
+     *
+     * @param filename  file name string of file to be written
+     * @param lines list of lines to be written to the file
+     * @throws UncheckedIOException wrapping an IOException if an io error occurs.
+     */
+    public static void writeLines(String filename, List<String> lines) {
+        Objects.requireNonNull(filename, "filename must not be null");
+        Objects.requireNonNull(lines, "lines must not be null");
+        writeLines(Path.of(filename), lines);
+    }
+
+    /**
+     * Write a list of strings as the content of the file at {@code path}.
+     * For example:
+     * {@snippet lang="java":
+     * var supply = List.of("apple", "pear", "orange");
+     * var path = Path.of("data.txt");
+     * writeLines(path, supply); // @highlight substring="writeLines"
+     * var lines = readLines(path);
+     * printlines(lines);
+     * }
+     * will print on the output stream as:
+     * {@snippet lang="text":
+     * apple
+     * pear
+     * orange
+     *
+     * }
+     *
+     * @implSpec Each line will be followed by the platform line separator.
+     *
+     * @param path  file name string of file to be written
+     * @param lines  list of lines to be written to the file
+     * @throws UncheckedIOException wrapping an IOException if an io error occurs.
+     */
+    public static void writeLines(Path path, List<String> lines) {
+        Objects.requireNonNull(path, "path must not be null");
+        Objects.requireNonNull(lines, "lines must not be null");
+        try (PrintWriter out = new PrintWriter(path.toFile())) {
```

Comment:
I assume this can use Files.write(path, lines).

Reply:
Thx

Reply:
Also, it could be named `writeLines` instead of `write` as the signature will disambiguate (which cannot be done for `readLines` vs. `read`).